client/src/components/NationwideMap.vue

```diff
@@ -285,6 +301,13 @@ export default defineComponent({
     routerLayer: function(newLayer: string) {
       this.setDataLayerVisibility(newLayer, true);
     },
+    // Listen for changes to lat/long to update map location.
+    'geoState.geoids.lat': function() {
+      this.zoomToLongLat();
+    },
+    'geoState.geoids.long': function() {
```

Comment:
You cracked the code for nested listening!!

Reply:
another vue quirk!